tools/serverpod_cli/test/generator/dart/code_generator_dart/static_comment_test.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:serverpod_cli/src/generator/dart/code_generator_dart.dart';
+import 'package:test/test.dart';
+
+import '../../../_test_util/builders/class_definition_builder.dart';
+import '../../../_test_util/builders/generator_config_builder.dart';
```

Comment:
Suggestion: Use package imports.

Even though dart suggests the usage of [relative imports](https://dart.dev/effective-dart/usage#prefer-relative-import-paths) in the style guide I think we [prefer path imports at the moment](https://github.com/serverpod/serverpod/issues/1146).